charts/operator/chart_test.go

```diff
@@ -234,6 +234,10 @@ const preTranspilerChartVersion = "0.4.28"
 // TestChartDifferences can be removed if in the next operator chart version values definition changes or any resource.
 // That test only validates clean transition to gotohelm definition of the operator helm chart.
 func TestChartDifferences(t *testing.T) {
```

Comment:
I think we're safe to remove this test now as we've finished the transition to gotohelm.

Think it would be worth it to add a test that runs Kustomize against the `appVersion` of the operator in the chart and checks for divergences of RBAC?

Reply:
Agree. It was just for me to validate I'm not breaking chart all over the places.

Reply:
👍 another reason I'd _love_ to merge some of this into the same codebase 😅

Reply:
Come to think of it, the operator chart might be a great first attempt at having the helm charts just follow another repository given that all the sources of truth at in the operator repo.